Assets/_Project/Scripts/Map/LocationFactory.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Mapbox.Unity.Location;
+using Mapbox.Unity.Map;
+using PolSl.UrbanHealthPath.Navigation;
+using UnityEngine;
+
+namespace PolSl.UrbanHealthPath.Navigation
+{
+    public enum LocationFactoryMode
+    {
+        Fake,
+        Device
+    }
+    public class LocationFactory : MonoBehaviour
```

Comment:
Again, why MonoBehaviour?

Reply:
This isn't really a factory as it doesn't create new instances. If the other classes weren't MonoBehaviours, you could just return new instances.

Reply:
Done.